megamolbart/megamolbart/inference_perceiver.py

```diff
@@ -0,0 +1,619 @@
+#!/usr/bin/env python3
+
```

Comment:
Can you add short docstrings with description of functions?
The use cases of some functions is not straight forward.